### PR TITLE
SQL fix and addressing feedback from demo

### PR DESCRIPTION
**Pull request by chenriksson**
Includes the following, with screenshots:
- Update SQL to return and not migrate when no request match

- Add informational hint on transform page to indicate what is expected for administrator:

image

- Add informational message when prompting for new admin sign in:

image

- Adjust transformation error formatting to align with header:

image

//cc @anangaur , @karann-msft

### CODE REVIEW

File: src/NuGetGallery/Controllers/AuthenticationController.cs [hunk 1]
```diff
@@ -62,10 +62,11 @@ public partial class AuthenticationController
         /// Sign In\Register view
         /// </summary>
         [HttpGet]
-        public virtual ActionResult LogOn(string returnUrl)
+        public virtual ActionResult LogOn(string returnUrl, string returnUrlMessage = "")
```
Comment (chenriksson) [inline src/NuGetGallery/Controllers/AuthenticationController.cs]: Just realized I'm putting this in url; and should probably move to a header. Will update in a few minutes...

Comment (joelverhagen) [inline src/NuGetGallery/Controllers/AuthenticationController.cs]: Why not `TempData`? Seems a bit weird to encode an entire message in the URL.

Comment (joelverhagen) [reply src/NuGetGallery/Controllers/AuthenticationController.cs]: Actually, in light of `returnUrl` it kind of makes sense... I'm torn 😄

File: src/NuGetGallery/Views/Users/TransformFailed.cshtml [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
+    <div class="row col-sm-8">
+        @if (TempData.ContainsKey("TransformError"))
+        {
+            <h1>Failed to transform to an organization</h1>
             <hr />
-            @TempData["TransformError"]
-        </div>
-    }
+            <div>
+                @TempData["TransformError"]
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/TransformFailed.cshtml]: Why is this `TempData` and not part of a view model?

File: src/NuGetGallery/Controllers/AuthenticationController.cs (hunk 1 above)
Comment (chenriksson) [reply src/NuGetGallery/Controllers/AuthenticationController.cs]: Agree it should not have been in URL. Fixed by moving to TempData.

File: src/NuGetGallery/Views/Users/TransformFailed.cshtml (hunk 2 above)
Comment (chenriksson) [reply src/NuGetGallery/Views/Users/TransformFailed.cshtml]: Laziness... fixed by creating strong type viewModel for error message.

**Comment by chenriksson** [comment]
@joelverhagen Addressed feedback. Also, reminder to review the screenshots... in case you haven't.

File: src/NuGetGallery/Controllers/UsersController.cs [hunk 3]
```diff
@@ -153,9 +151,14 @@ public virtual ActionResult TransformToOrganization()
             
             await _userService.RequestTransformToOrganizationAccount(accountToTransform, adminUser);
 
-            // prompt for admin sign-on to confirm transformation
+
```
Comment (scottbommarito) [inline src/NuGetGallery/Controllers/UsersController.cs]: nit: 2 lines whitespace here instead of one?

File: src/NuGetGallery/Strings.resx [hunk 4]
```diff
@@ -734,6 +734,9 @@ For more information, please contact '{2}'.</value>
     <value>Administrator account '{0}' has not confirmed their email address.</value>
   </data>
   <data name="TransformAccount_RequestExists" xml:space="preserve">
-    <value>Another tranform request was created on {0} with organization admin '{1}'.</value>
+    <value>Another tranform request was created on {0} for administrator '{1}'. Note that a new request will overwrite this.</value>
```
Comment (shishirx34) [inline src/NuGetGallery/Strings.resx]: ... this request.

**Pull request merged**